StripeCore/StripeCore/Source/Connections Bindings/ConnectionsSDKInterface.swift

```diff
@@ -20,6 +20,7 @@ import UIKit
         apiClient: STPAPIClient,
         clientSecret: String,
         returnURL: String?,
+        onEvent: ((FinancialConnectionsEvent) -> Void)?,
```

Comment:
`FinancialConnectionsSDKInterface` is internal; all the interfaces accessible by merchants should be optional (though please do correct me if you see otherwise). In other words, there should be no API-breaking changes. Note that Financial Connections is also in beta so even if this was not true, we have some leeway into making changes anyway.

Reply:
hmm so iuuc `FinancialConnectionsSDKInterface` is not public and shouldn't be accessed by merchants right?

Reply:
oh sorry, I see the confusion

```
@_spi(STP) public protocol FinancialConnectionsSDKInterface {
```

The `@_spi(STP)`, or specifically `@_spi`, makes this only "public" _If_ someone imports it by using `@_spi(STP)`

It's essentially "not public," _unless_ someone adds `@_spi(STP)` to the import (which would not be recommended).